app/app.go

```diff
@@ -0,0 +1,5 @@
+package app
+
+const (
+	AppVersion = "v0.0.1-pre-alpha.1" // DISCUSS (team): see app/pocket/main.go, one of the two should be a source of truth, potentially replaced at build-time as described in `docs/build/README.md`
```

Comment:
since `main` depends on any and everything else, directly or indirectly, version number should be provided by a non-`main` package. I suspect this should be the source of truth.

Corollary: this `package` shouldn't use many (or any?) packages from the rest of the app so that anyone can import the version information.

Reply:
I have updated accordingly and also added some pointers on how to override this at build time:

![image](https://user-images.githubusercontent.com/29378614/201209200-f1ccbf50-e8e5-481a-af26-efab32c10b15.png)

Useful if we also want to tag with the git-commits our official releases later on.

https://github.com/pokt-network/pocket/pull/176/commits/5ce35632525be1e610787d710b83640b5633da62

@andrewnguyen22 PTAL if you are happy about this